docs/timelines/timeline-options.md

```diff
@@ -0,0 +1,104 @@
+---
+id: timeline-options
+title: Timeline Options
+sidebar_label: Timeline options
+---
+
+Timelines have a number of options which set how they operate. 
+Press \<Options\> (or the \[Options\] softkey on the top level menu) then the **select** button for the timeline you wish to edit.
+
+--- 
+
+## Handle Tab
+
+All options are the same as for a cue, see [Options](../cues/playback-options.md#handle-tab).
```

Comment:
```suggestion
All options are the same as for a memory, see [Options](../cues/playback-options.md#handle-tab).
```

Reply:
That is an interesting one. Everywhere else in the manual a single-state look is called a "cue". And the Options screen calls them "cue 20" (from the default legend which is Cue n). But if you press <Record> then the first option is [Create Memory]. So what is the correct term to use?

Reply:
I agree it's an ongoing source of frustration, and there isn't a satisfactory solution without us removing some of the inconsistency in the UI (the default legend for a memory would be a great place to start, filed #87511). The problem with referring to a memory *as* a cue is that it's then ambiguous as to what the word "cue" means - does it mean the thing that lives within a chase/cuelist, or a memory (a "single-cue playback")? Within the software, a cue is a completely separate type of object to a playback - cues exist strictly *within* playbacks, and they have different properties, for example.

My general feeling is that we should settle on playbacks being one of three types; a memory, a cuelist or a chase. Cuelists and chases contain more than one cue, whereas "memory" is sort of a shorthand for a "single-cue playback". That might be out of the scope of this PR though if we currently use Cue everywhere! Don't know what others think (@AvolitesLtd/titan ?)

Reply:
From a user perspective, without internal knowledge of the software, I would say calling it a "cue" makes for easier understanding than calling it a "memory".  Then the term "cue list" makes sense as it appears to be a list of single cues (we try in the manual to refer to "steps" in a chase rather than "cues" to avoid that confusion). 

This is only an opinion from myself and a small range of Titan-using friends though, so your wider experience of Titan users may lead to a different answer.

Reply:
Yeah agreed, I think there is some inconsistency in the software itself. I don't think we should worry about it for this PR and rather address the inconsistency more widely if we feel its needed.

Reply:
OK I will leave it saying "cue" here for now